packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -0,0 +1,128 @@
+import retrying
+
+
+LATENCY = 60
+
+
+@retrying.retry(wait_fixed=2000, stop_max_delay=LATENCY * 1000)
```

Comment:
@malnick can you address this question? I see we still have some retrying logic in the PR / not sure if this was ever addressed or discussed.